command/alloc_exec.go

```diff
@@ -38,7 +38,7 @@ Usage: nomad alloc exec [options] <allocation> <command>
   When ACLs are enabled, this command requires a token with the 'alloc-exec',
   'read-job', and 'list-jobs' capabilities for the allocation's namespace. If
   the task driver does not have file system isolation (as with 'raw_exec'),
-  this command requires the 'alloc-node-exec', 'read-job', and 'list-jobs'
+  this command requires the 'alloc-node-exec', 'alloc-exec', 'read-job', and 'list-jobs'
```

Comment:
Note: in testing this out, noticed that `raw_exec` jobs required both `alloc-exec` AND `alloc-node-exec` capabilities. Updating the docs to reflect.